docs/UserGuide.md

```diff
@@ -433,8 +434,8 @@ managers to keep track of the rooms that are cleaned or uncleaned for the day.
 
 This command clears **all entries** from GuestBook. 
 
-This command is typically used as a **hard reset** of all guest data in GuestBook. You should not be using
-this command unless you want to star the GuestBook on a clean slate.
+This command is typically used as a **hard reset** of all guest data in GuestBook. You **should not** be using
```

Comment:
Should this be a warning call out? Maybe bangbang like in line 160?

Reply:
There is a warning given below already!